teamworkApp/lib/evaluateAnswers.py

```diff
@@ -3,47 +3,39 @@
 
 DB = 'db/development.sqlite3'
 
+QUESTIONS = [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17]
```

Comment:
I am somewhat confused about why this is a list from 0 to 17. Wouldn't it be simpler to have a variable that is number of questions and then use range if you need to iterate from 0 to one less than that number?

Reply:
Oh, this seems to be a vestige of when I was going through your implementation and you seemed to be getting a list from the database that would always be [0..17] so I put this in place of that line and forgot to switch it after I refactored. I will refactor this.